lib/improver/tests/nbhood/use_nbhood/test_CollapseMaskedNeighbourhoodCoordinate.py

```diff
@@ -71,6 +71,21 @@ class Test_reweight_weights(IrisTest):
 
     def setUp(self):
         """Set up a cube."""
+        self.mask = np.array([[[1, 1, 1., 0, 0],
```

Comment:
There's still a "1." here!

Reply:
Done a lot of tidying up. Hopefully they are fine now.